go.mod

```diff
@@ -1,3 +1,64 @@
 module github.com/xmtp/xmtpd
 
 go 1.21.11
```

Comment:
We should switch to go 1.22, since we are starting fresh and that's the current stable version